www/src/routes/Landing.tsx

```diff
@@ -0,0 +1,214 @@
+                <Intro />
+                <Values />
+                <Services />
+            </Grid>
+        </Grid>
+    );
+};
+
+const Intro: FC = () => {
+    const classes = useStyles();
+
+    return (
+        <Grid container item>
+            <Grid container item style={{ height: '100vh' }}>
+                <Grid container item xs={5} alignItems='center'>
+                    <Grid item>
+                        <Box maxWidth='sm'>
+                            <img src={BlackLogo} className={classes.logo_1} />
+                        </Box>
+                        <Typography variant='h1'>CompE+</Typography>
+                        <Typography variant='h3'>Practice, Insight, and Support</Typography>
+                    </Grid>
+                </Grid>
+                <Grid container item xs={7} alignItems='center' justify='center'>
+                    <Box maxWidth='sm'>
+                        <img src={PairProgramming} className={classes.svg} />
+                    </Box>
+                </Grid>
+                <Grid container item xs={12} justify='center'>
+                    <Grid container item alignItems='center' justify='center' direction='column' spacing={3}>
+                        <Grid item>
+                            <Typography component='h2' variant='h2' align='center'>
+                                CompE+ is a student-led pilot program to provide support for computer engineering students looking for internships
+                            </Typography>
+                        </Grid>
+                        <Grid item>
+                            <OutlinedButton title={'Get Started'} url={'#'} />
+                        </Grid>
+                    </Grid>
+                </Grid>
+            </Grid>
+            <Grid container item spacing={10}>
+                <Grid container item xs={8} alignItems='center' justify='center'>
+                    <Grid item>
+                        <Typography component='h2' variant='h1' noWrap>
+                            What’s CompE+ ?
+                        </Typography>
+                        <Typography component='h2' variant='body1'>
+                            CompE+ is a student led pilot program to provide CompE students with practice, insight, and support when finding internships in the tech industry. Arguably, computer
+                            engineering is a unique discipline and requires different resources and support - that’s where CompE+ comes in!{' '}
+                        </Typography>
+                    </Grid>
+                </Grid>
+                <Grid container item xs={4} alignItems='center' justify='center'>
+                    <img src={BlackLogo} className={classes.logo_2} />
+                </Grid>
+            </Grid>
+        </Grid>
+    );
+};
+
+const Values: FC = () => {
+    const classes = useStyles();
+
+    return (
+        <Grid container item spacing={8}>
```

Comment:
Nit the outermost `Grid` should only use `container` since it's not an item and terminal Grid elements should only have `item`. Take a look at [Nested Grid](https://material-ui.com/components/grid/#nested-grid)

Reply:
oh it is an item tho (I think). I split up the Landing into 3 main parts: Intro, Values, and Services. In line 26 it shows that they're nested Grids so I add the `item` field to the subparts of the Landing page. 

The only reasoning for separating into separate sections was it was easier for me to look at a smaller chunk than the whole thing, and allows us to space sections apart easier if necessary.

Reply:
Ah my bad yes this is good 👍